docs/installation.rst

```diff
@@ -84,41 +82,66 @@ There are no **mandatory** external dependencies. However, some optional feature
 .. note:: You can install these additional components at any time -- before or after installing PyMuPDF. PyMuPDF will detect their presence during import or when the respective functions are being used.
 
 
-Installation from source without using an sdist
+Build and install from local PyMuPDF checkout and optional local MuPDF checkout.
 ---------------------------------------------------------
 
-* First get a PyMuPDF source tree:
+* Install C/C++ development tools as described above.
+
+* Enter a Python venv and update pip, as described above.
+
+* Get a PyMuPDF source tree:
 
-  * Clone the git repository at https://github.com/pymupdf/PyMuPDF,
-    for example::
+  * Clone the PyMuPDF git repository:
 
-      git clone https://github.com/pymupdf/PyMuPDF.git
+        git clone https://github.com/pymupdf/PyMuPDF.git
 
   * Or download and extract a `.zip` or `.tar.gz` source release from
     https://github.com/pymupdf/PyMuPDF/releases.
 
-* Install C/C++ development tools and SWIG as described above.
 
-* Build and install PyMuPDF::
+* Build and install PyMuPDF:
 
-    cd PyMuPDF && python setup.py install
+      cd PyMuPDF && pip install .
 
   This will automatically download a specific hard-coded MuPDF source release,
   and build it into PyMuPDF.
-  
+
+
+Build and install PyMuPDF using a local MuPDF source tree:
+
+* Clone the MuPDF git repository:
+
+      git clone --recursive https://ghostscript.com:/home/git/mupdf.git
+
+*
+  Build PyMuPDF, specifying the location of the local MuPDF tree with the
```

Comment:
I think the text should start after the `*` bullet.

Reply:
I sometimes do it this way so i can easily word-wrap the text. It does look less good as plain text, but realistically i suspect no one actually reads docs by looking at the plain markup.

Reply:
fair enough